content/zh/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md

```diff
@@ -58,32 +60,31 @@ build` can still run in your Kubernetes cluster.
 Docker 仍然是构建容器的利器，使用命令 `docker build` 构建的镜像在 Kubernetes 集群中仍然可以运行。
 
 <!-- 
-If you’re using a managed Kubernetes service like GKE, EKS, or AKS (which [defaults to containerd](https://github.com/Azure/AKS/releases/tag/2020-11-16)) you will need to
+If you’re using a managed Kubernetes service like AKS, EkS or GKE, you will need to
 make sure your worker nodes are using a supported container runtime before
 Docker support is removed in a future version of Kubernetes. If you have node
 customizations you may need to update them based on your environment and runtime
 requirements. Please work with your service provider to ensure proper upgrade
 testing and planning. 
 -->
-如果你正在使用 GKE、EKS、或 AKS 
-([默认使用 containerd](https://github.com/Azure/AKS/releases/tag/2020-11-16))  
-这类托管 Kubernetes 服务，你需要在 Kubernetes 后续版本移除对 Docker 支持之前，
+如果你正在使用 GKE、EKS、或 AKS 这类托管 Kubernetes 服务，
```

Comment:
Should be `AKS、EkS、或 GKE`.

Reply:
EKS 应该不算错吧

Reply:
Yep, it should be `EKS` instead of `EkS`. I open another PR to fix it:
- #33709

The English version changed from `GKE, EKS, AKS` to `AKS, EKS, GKE`, I think Mandarin version should follow too.